### ISSUE TITLE
Commandbar elements within detailslist not accessible

### ISSUE THREAD

**Issue by bhuvan-iyer** [OfficeDev/office-ui-fabric-react#10119]
### Environment Information

- **Package version(s)**: 7.20.1
- **Browser**: Chrome, Edge 
### Describe the issue:

Commandbar elements when within detailslist is not accessible using tab stops or arrow keys. But a button inside detailslist is accessible. 

#### Expected behavior:

Commandbar elements when within detailslist should be accessible using tab stops or arrow keys. 

### codepen repro:

https://codepen.io/kbh_Ms/pen/mNzxvO

### Priorities and help requested :

Requested priority: High

Products/sites affected: Internal Microsoft application

<img width="832" alt="accessibility_bug_office_fabric" src="https://user-images.githubusercontent.com/53990751/62834538-b1155a80-bc6b-11e9-805c-80f2f5af3d1e.PNG">

**Comment by JasonGore** [comment]
I see the behavior described, thanks for the CodePen! Both `CommandBar` and `DetailsRow` contain a `FocusZone` which makes this a nested scenario. There could either be an issue with nested `FocusZones` or a configuration issue.

**Comment by bhuvan-iyer** [comment]
Hi,

What is the status of this bug. Is it fixed ?

**Comment by joschect** [comment]
@bhuvan-iyer I don't believe that this is a bug any more. The commandbar really isn't supposed to be used within a detailslist. Per our earlier conversation there is [this codesandbox](https://codesandbox.io/s/keen-water-gtymy) that gives you all the benefits of a commandbar by implementing a composed version. It's relatively small and is your best bet moving forward.

**Comment by bhuvan-iyer** [comment]
ok.

**Comment by JasonGore** [comment]
Please let us know if you have any other questions or comments. I will close this issue as "By Design". Thanks!
